pkg/aws/core_test.go

```diff
@@ -289,6 +301,22 @@ var _ = Describe("MachineServer", func() {
 					errMessage:        "machine codes error: code = [Internal] message = [Region doesn't exist while trying to create session]",
 				},
 			}),
+			Entry("Placement object with all fields set", &data{
```

Comment:
Is there a way to check if the placement group was set on the AWS machine create request?